distribution/openrct2.d.ts

```diff
@@ -2429,6 +2429,11 @@ declare global {
         remove(): void;
     }
 
+    /**
+     * Vehicle pitch is not an incremental angle. See VehicleData.cpp AccelerationFromPitch for more information about the values.
+     */
+    type CarPitch = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15 | 16 | 17 | 18 | 19 | 20 | 21 | 22 | 23 | 24 | 25 | 26 | 27 | 28 | 29 | 30 | 31 | 32 | 33 | 34 | 35 | 36 | 37 | 38 | 39 | 40 | 41 | 42 | 43 | 44 | 45 | 46 | 47 | 48 | 49 | 50 | 51 | 52 | 53 | 54 | 55 | 56 | 57 | 58 | 59;
```

Comment:
Also are we sure these are all of them? Didn't X7 add more sprite angles for his various new track elements?

Reply:
X123M3-256 did not add any new angles, he added paint code for new combinations of existing pitch and bank angles.

Reply:
For the record, `bankRotation` is not incremental either.

Reply:
> X123M3-256 did not add any new angles, he added paint code for new combinations of existing pitch and bank angles.

I didn't add new angles but doing so is very simple, I actually had a test branch where I did that. IIRC the new angle just has to be added to a few arrays. I don't really know what this PR is about but I think if it is possible to do so, it would be worth accounting for the possibility that new angles might be added down the line.

Reply:
I don't see the advantage of adding this type, there are also other places in the API that have restricted parameters where we did not add a type. In general, number unions are not always helpful, i.e. even if `x` is known to be natural, `x % 60` will not be recognised by typescript as `CarPitch`.

Reply:
This union was removed